### PR TITLE
[1.21.0] release notes

### PR DESCRIPTION
**Pull request by salahalsaleh**

### CODE REVIEW

File: content/release-notes/1.21.0.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+date: 2020-10-23
+linktitle: "1.21.0"
+title: "1.21.0"
+kubernetes: "1.17, 1.18, and 1.19"
+---
+
+{{<changes>}}
+* Support bundle files will now be saved under a subdirectory, so that it won't pollute the current directory when extracted.
+* Rolling back a version with known invalid files will show a "Invalid files" modal first before deploying.
```
Comment (laverya) [inline content/release-notes/1.21.0.md:10]:
```suggestion
* Rolling back to a version with known invalid files will show a "Invalid files" modal first before deploying.
```
maybe?

File: content/release-notes/1.21.0.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+{{</changes>}}
+
+{{<fixes>}}
+* Fixed a bug that caused rolling back a previously failed deploy to still show the status as "failed" after a successfull rollback.
```
Comment (laverya) [inline content/release-notes/1.21.0.md:14]:
```suggestion
* Fixed a bug that caused rolling back to a previously failed deploy to still show the status as "failed" after a successful rollback.
```

File: content/release-notes/1.21.0.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
+date: 2020-10-23
+linktitle: "1.21.0"
+title: "1.21.0"
+kubernetes: "1.17, 1.18, and 1.19"
+---
+
+{{<changes>}}
+* Support bundle files will now be saved under a subdirectory, so that it won't pollute the current directory when extracted.
+* Rolling back to a version with known invalid files will show a "Invalid files" modal first before deploying.
```
Comment (markpundsack) [inline content/release-notes/1.21.0.md:10]:
```suggestion
* Rolling back to a version with known invalid files will show an "Invalid files" modal first before deploying.
```

Comment (markpundsack) [reply content/release-notes/1.21.0.md:10]: Maybe "confirmation modal"? Showing a modal seems kind of weird, like that doesn't solve anything. Maybe what you really mean is "rolling back to a version with known invalid files will require confirmation before deploying"

Comment (markpundsack) [reply content/release-notes/1.21.0.md:10]:
```suggestion
* Rolling back to a version with known invalid files will require confirmation before deploying.
```

File: content/release-notes/1.21.0.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+{{</changes>}}
+
+{{<fixes>}}
+* Fixed a bug that caused rolling back to a previously failed deploy to still show the status as "failed" after a successful rollback.
```
Comment (markpundsack) [inline content/release-notes/1.21.0.md:14]: This sentence is rather hard to parse. Can we simplify?

**Review by markpundsack — APPROVED** [review]
